effectsmanager.h

```diff
@@ -257,9 +257,6 @@ class EffectsManager : public IEffectsManager
     {
         return _currentEffectIndex >= 0 && _currentEffectIndex < static_cast<int>(_effects.size());
     }
-
-    friend void to_json(nlohmann::json &j, const EffectsManager &manager);
-    friend void from_json(const nlohmann::json &j, EffectsManager &manager);
```

Comment:
These friend function declarations referred to nothing, because the EffectsManager JSON functions (de)serialize an IEffectsManager. Also and more relevantly they were unnecessary because to_json() and from_json() use public getter functions and the constructor, respectively.